pkg/network/master/egressnp.go

```diff
@@ -0,0 +1,47 @@
+package master
```

Comment:
can you rename this file to `egress_network_policy.go` so it matches the one in `pkg/network/node`?